### PR TITLE
home page and some about page

### PR DESCRIPTION
**Pull request by axelKeizoStahl**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -2,122 +2,149 @@ import Head from 'next/head'
...
+  const [faq1, setfaq1] = useState(1)
+  const [faq2, setfaq2] = useState(2)
+  const [faq3, setfaq3] = useState(3)
+  const [faq4, setfaq4] = useState(3)
+  const [faq5, setfaq5] = useState(3)
+  const [faq6, setfaq6] = useState(3)
+  const [faq7, setfaq7] = useState(3)
+  const handleClick = (id: string) => {
+    switch(id) {
+      case "news1":
```
Comment (TheEgghead27) [inline src/pages/index.tsx:26]: What

File: src/pages/index.tsx [hunk 2]
```diff
@@ -2,122 +2,149 @@ import Head from 'next/head'
...
+  function switchstack1(j) {
+    switch (j) {
+      case 'faq4':
+        setfaq4(2);
+        break;
+      case 'faq5':
+        setfaq5(1);
+        break;
+      case 'faq6':
+        setfaq6(2);
```
Comment (TheEgghead27) [inline src/pages/index.tsx:75]: w ha t

File: src/pages/about.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+export default function About() {
+
+  var prezdescription: string = "";
+
+  const [vannaOnTop, setVannaOnTop] = useState(false);
+  const [caylaOnTop, setCaylaOnTop] = useState(false);
+
+  const handleClick = (id: string) => {
+    if (id === 'vanna') {
+      setVannaOnTop(true);
```
Comment (TheEgghead27) [inline src/pages/about.tsx:20]: :sob:

File: src/styles/globals.css [hunk 4]
```diff
@@ -1,107 +1,248 @@
...
+  top: 60px;
+  position: relative;
+  background-color: #CCCCCC;
+  height: 150px;
+  width: 300px
+}
+#upcoming {
+  text-align: center;
+}
+#news1 {
```
Comment (TheEgghead27) [inline src/styles/globals.css:108]: my brother in css 

classes

File: src/pages/index.tsx (hunk 1 above)
Comment (axelKeizoStahl) [reply src/pages/index.tsx:26]: Lmao, I've had no time this vacation, needed to get started, gonna clean up and revise later

File: src/styles/globals.css (hunk 4 above)
Comment (TheEgghead27) [reply src/styles/globals.css:108]: especially if you compartmentalize things into components

**Review by TheEgghead27 — COMMENTED** [review]
yes i agree

File: src/pages/index.tsx (hunk 1 above)
Comment (TheEgghead27) [reply src/pages/index.tsx:26]: gl hf

File: src/styles/globals.css (hunk 4 above)
Comment (axelKeizoStahl) [reply src/styles/globals.css:108]: Why

Comment (TheEgghead27) [reply src/styles/globals.css:108]: easy code reuse without copy-paste

Comment (axelKeizoStahl) [reply src/styles/globals.css:108]: Why classes I mean

Comment (TheEgghead27) [reply src/styles/globals.css:108]: css classes literally do that

Comment (axelKeizoStahl) [reply src/styles/globals.css:108]: That's better than ids cause classes can be many things ok

Comment (axelKeizoStahl) [reply src/styles/globals.css:108]: Also, I pr'ed like 5 seconds ago how r u here

Comment (TheEgghead27) [reply src/styles/globals.css:108]: ;)

Comment (axelKeizoStahl) [reply src/styles/globals.css:108]: Damn GitHub copilot is so good it predicted everything

**Pull request closed without merging**